moped-editor/src/views/projects/projectView/ProjectComponents/ComponentMapToolbar.js

```diff
@@ -14,6 +14,9 @@ const useStyles = makeStyles((theme) => ({
   appBar: {
     zIndex: theme.zIndex.drawer + 1,
   },
+  toolbarContainer: {
+    backgroundColor: "#fff",
+  },
```

Comment:
🙌

Reply:
Tiny nitpick, but maybe this could be a new entry in the MUI theme in `src/theme/index.js`? Also acknowledging that this isn't the only place in the code where we use hex codes outside that file!

Reply:
not a nitpick! fixed it.